docs/curl.1

```diff
@@ -1552,8 +1552,10 @@ otherwise normally resolved address to be used. Consider it a sort of
 /etc/hosts alternative provided on the command line. The port number should be
 the number used for the specific protocol the host will be used for. It means
 you need several entries if you want to provide address for the same host but
-different ports.
-
+different ports. The definitions from --resolve supersede /etc/hosts and
+network name resolution mechanisms. this option will not disable DNS name
+resolutions, so --resolve host:port:address just ensures that when curl
```

Comment:
Well, it _does_ prevent the DNS from being used for the provided host name so I think this wording adds confusion.